app/src/main/java/org/p2p/wallet/user/repository/UserInMemoryRepository.kt

```diff
@@ -39,4 +69,28 @@ class UserInMemoryRepository : UserLocalRepository {
 
         return data
     }
+
+    private fun getPopularDecimals(): List<TokenData> {
+        var items = mutableListOf<TokenData>()
+        var tokens = decimalsFlow.value
+        for (symbol in popularItems) {
+            try {
+                val token = tokens.first { it.symbol == symbol }
+                items.add(token)
+            } catch (e: NoSuchElementException) {
```

Comment:
```suggestion
            val token = tokens.firstOrNull { it.symbol == symbol }
            if (token != null) items.add(token)
```

И можно убрать try catch